src/LiveComponent/src/DependencyInjection/LiveComponentExtension.php

```diff
@@ -48,7 +65,6 @@ public function load(array $configs, ContainerBuilder $container): void
             ->setArguments([
```

Comment:
On the previous line, I just noticed we're registering some services with the class as their id. I'd like to avoid that - it makes them autowireable.

Reply:
Ah good point. I will change in both bundles.

Reply:
Done.